### ISSUE TITLE
[Bug Report][1.5.16] v-menu trigger issue with v-text-field box

### ISSUE THREAD

**Issue by angelpcadena** [vuetifyjs/vuetify#7610]
### Versions and Environment
**Vuetify:** 1.5.16
**Vue:** 2.6.10
**Browsers:** Chrome 75.0.3770.100
**OS:** Windows 10

### Steps to reproduce
Try to trigger the v-menu while clicking on the edges of the boxed text field, the text field "activates" but the menu is not shown.

### Expected Behavior
For the menu to show when it's field is activated

### Actual Behavior
The field activates but the menu doesn't appear

### Reproduction Link
<a href="https://codepen.io/anon/pen/rEjYMP?editors=1010" target="_blank">https://codepen.io/anon/pen/rEjYMP?editors=1010</a>

**Comment by johnleider** [comment]
Updated codepen for v2.2: https://codepen.io/johnjleider/pen/VwYygRG

**Comment by adambiggs** [comment]
No plans to fix this for v2? Does anybody have a workaround in the meantime?

**Comment by johnleider** [comment]
I believe this is solved for v3, @KaelWD will have to confirm. However, I don't believe this is going to be resolved in v2.

**Comment by adambiggs** [comment]
@KaelWD if this is fixed in v3 already, any chance you could point me to the PR/commit where it was fixed?

I'd like to find a workaround for v2 while we wait for v3 to become production ready. I'll be sure to post here if I do.

**Comment by johnleider** [comment]
I should have been more clear. It's solved in v3 in the sense that it won't be an issue "once" we implement it. Inputs are actively being worked on as a part of [Phase 3](https://github.com/vuetifyjs/vuetify/issues?q=is%3Aopen+is%3Aissue+label%3A%22Vuetify+3+-+Phase+3%22).

**Comment by adambiggs** [comment]
Ah that makes sense, thanks for clarification!

**Comment by adambiggs** [comment]
I found a simple CSS workaround:

```css
.v-text-field__slot > .v-label {
    pointer-events: none;
}
```

Hopefully this helps the next person who finds themselves here :)

**Comment by jamesdruhan** [comment]
I've found a better workaround for **filled v-text-field** that enables the date menu to work perfectly around the entire field.

https://codepen.io/jamesdruhan/pen/qBjxpza

When using the v-text-field (filled) with a date picker, add a custom class like so: 

```js
<v-text-field  v-model="date" filled label="Picker in menu" class="vTextFilled_dateFix" v-on="on"></v-text-field>
```

```css
.vTextFilled_dateFix {
  input {
    margin-top: 0px !important;
    padding-top: 28px !important;
    max-height: 100% !important;
  }
}
```
The padding-top pixel value changes depending on your field height (ie. if dense is used or not). A bit manual but it works perfectly.

**Comment by bjorntrondsen** [comment]
Spinning off of jamesdruhan's answer, I had to do the following in vuetify 2.5:
```
.vTextFilled_dateFix
  input
    margin-top: 0px !important
    max-height: 100% !important
  label
    pointer-events: none !important

```

**Comment by KaelWD** [comment]
It isn't fixed in v3 and for the same reason, all events are passed to the `<input>` element instead of the root. Clicking on the box outside of the input element calls `.focus()` on the input, but the menu isn't shown because it listens for click.

**Comment by KaelWD** [comment]
Fixed by #15904
